### PR TITLE
Enable more authors in RSS template

### PR DESCRIPTION
**Pull request by dmi3kno**
This is a breaking change with regards to `article_rss()`. I needed more authors and optional email field, so I decided to refactor this template a bit. The class used by RSS has some known issues with formatting table captions. 
https://tex.stackexchange.com/questions/322231/caption-makes-table-move-in-statsoc-class
The trick is to remove .aux file before re-rendering. I wonder if we should write this somewhere in the documentation.

### CODE REVIEW

**Review by cderv — CHANGES_REQUESTED** [review]
Thanks! 

Regarding the breaking change, as the fields don't have the same names, we could probably support both in the template (like `$if(author)$` would be the old way and `$if(authors)$` the new way) 

Regarding the table bug, we can also not support Pandoc's table (at least the one that does not work) inside this format. I am not quite sure to have fully comprehend the issue, do you have a reproducible example ? 
If we need to do something special (like a patch or something) we can do that in the format function (as post processor or something else). Removing aux file and re-rendering is maybe possible but it seems a pretty hard hack. 

I just need to understand what is the issue really. It seems that `statsoc.cls` has not been updated since a long time.  

Small change needed also as we need to live preferably without a minimum pandoc version. 

Thanks

File: inst/rmarkdown/templates/rss/resources/template.tex [hunk 1]
```diff
@@ -18,19 +36,15 @@
...
-$if(author2)$
-\author[$short_author$]{$author2$}
-\address{$affiliation2$,
-$city2$,
-$country2$}
-\email{$email2$}
-$endif$
+
+$for(authors)$
+\author[$short_author$]{$it.name$}
```
[the remark this answers is not in the document]
Comment (dmi3kno) [reply inst/rmarkdown/templates/rss/resources/template.tex:39]: Yes, but that's how it was envisaged all along look at lines 21-33 in the code today. Besides it is not the same value. It's only the same index. The author name which will be displayed is specified in curly brackets.

File: inst/rmarkdown/templates/rss/resources/template.tex [hunk 2]
```diff
@@ -18,19 +36,15 @@
...
+\address{$it.affiliation$,
+$it.city$,
+$it.country$}
+$if(it.email)$\email{$it.email$}$endif$
```
[the remark this answers is not in the document]
Comment (dmi3kno) [reply inst/rmarkdown/templates/rss/resources/template.tex]: Agreed.

**Review by cderv — APPROVED** [review]
Thanks! 

I'll tweak a little and merge

**Pull request merged**